### ISSUE TITLE
Evolve the lenses / interface

### ISSUE THREAD

**Issue by jchalifour** [suchnsuch/Tangent#104]
A few things that occurred to me while trying to use the two existing lenses (cards and feed). I'll put these interface usage issues/suggestions here. Maybe it's better to separate these though. 

Version: 0.8.1
OS: Linux (probably isn't relevant to this suggestion)

1. Card lens: if my mouse hovers over a card, it scrolls in the card and seems a little difficult to get the pointer just right, in between cards to scroll the page. Maybe the gaps between cards could be bigger or some other method to make scrolling the page a little easier? 
2. Could cards display the YAML title field, optionally, instead of the file name as their main heading? Zettlr, for example, has a technique for this with a settings option to choose preference between filename, H1, or YAML title. Since we can already see the filename in the hierarchy, having the YAML title as a main source in the cards lens could be a useful thing.
3. Finding notes via cards: I struggle a bit to see what I'm looking for among all the cards, then recall that they are all listed in the hierarchy. So I go back and forth between the folder/file structure hierarchy and the card/feed lenses. There seems to be some slight dissonance in how I should use them. I'm wondering about ways to improve that browse/seek workflow. One thing struck me in [issue 97](https://github.com/suchnsuch/Tangent/issues/97) about manual sorting, perhaps having another lens that was a list style with options to sort differently would be a useful approach to both that issue and my instinct to use these lenses differently from the hierarchical folder/file list.

Thanks for considering this feedback.

**Comment by taylorhadden** [comment]
Having an option on whether or not cards should have scrollable contents might be the better solution. I've gone back and forth on what the behavior should be. I created #105 to represent this.

The notion of having some other title source has been brought up before. It doesn't look like there's an existing issue for it, so now it's #106.

Let's elaborate on item 3! In what context are you looking at the cards? A list lens is definitely a gap in the current lens system.

**Comment by jchalifour** [comment]
Thanks! I think since that card view can be very prominent I tend to look at it locate certain notes that I know exist within a certain folder. If I know I want to edit a specific note, I might click the folder and then look for it among the cards. I suppose this is especially true if it's not something linked from something I'm already working with.

**Comment by descendingnerdliness** [comment]
Can we set the default lens to list?

**Comment by taylorhadden** [comment]
Yeah, that's a pretty easy thing to pull off. Go ahead and make a separate ticket!
